src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java

```diff
@@ -61,8 +62,9 @@ boolean preconditionsAreMet(Loan loan, DateTime systemTime, Boolean shouldCountC
 
   private CompletableFuture<Result<Integer>> minutesOverdueExcludingClosedPeriods(Loan loan, DateTime returnDate) {
     DateTime dueDate = loan.getDueDate();
+    String ownerServicePoint = getOwnerServicePoint(loan).toString();
```

Comment:
If method is renamed, the variable should be renamed too.

Reply:
renamed